semantic/src/site/globals/site.overrides

```diff
@@ -11,8 +15,112 @@ p strong {
   font-weight: @semiboldWeight;
 }
 
-.ui.basebody.container {
-  background: rgb(248, 248, 248);
+#main-container {
+  padding: 0;
+  position: relative;
+  min-height: 100vh;
+  // Max width is 1027px, from Semantic-UI Container
+}
+
+#user-area {
+  font-family: @userAreaFont;
+  color: @userAreaText;
+  display: flex;
+  position: fixed; // TO-DO: No fixed on mobile
+  background-color: @headersInteractiveLow;
+  padding-top: 10px;
+  padding-bottom: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
+  height: @headerHeight; // TO-DO: Adjust for mobile
+  justify-content: space-between;
+  //   top: 0,
+  //   zIndex: 1000,
+}
+
+#user-area.ui.container {
+  margin-left: 0px !important;
+  margin-right: 0px !important;
+}
+
+#user-area-left {
+  display: flex;
+  flex-direction: column;
+}
+
+#menu-bar {
+  // border: 1px white solid;
+}
+// Menu bar links
+#user-area a {
+  color: @userAreaText;
+  font-size: 14px;
```

Comment:
It would be good to use a variable here instead. That was what I was aiming with this:
https://github.com/openmsupply/application-manager-web-app/blob/f14d65dc8707db2dd4a755cd2c0514bcf1ea37f9/semantic/src/site/globals/site.variables#L90-L110
But not sure exactly what should be the approach, anyway we want to use a variable here in case we need to change that globally we don't  have to find places using the style `font-size: 14px;` one by one.

Reply:
Yeah, sure -- maybe you could replace this in your branch since you're going to be updating these variables. ?

Reply:
Okay, for all these, I have put a comment `// REPLACE WITH VARIABLE` after them, so you can just search for that and replace them with the variables you've created in your branch.